Framework/Algorithms/test/GetDetOffsetsMultiPeaksTest.h

```diff
@@ -428,8 +424,8 @@ class GetDetOffsetsMultiPeaksTest : public CxxTest::TestSuite {
   /** Generate noisy data in a workspace
    */
   void generateNoisyData(MatrixWorkspace_sptr WS) {
-    Mantid::MantidVec &Y = WS->dataY(0);
-    Mantid::MantidVec &E = WS->dataE(0);
+    auto &Y = WS->mutableY(0);
+    auto &E = WS->mutableE(0);
     for (size_t i = 0; i < Y.size(); ++i) {
       Y[i] = static_cast<double>(rand() % 5);
```

Comment:
Wouldn't that add an additional loop, because currently both are assigned in one for loop, while otherwise they'd both have to loop through their data?